modules/swagger-parser/src/test/java/io/swagger/parser/SwaggerParserTest.java

```diff
@@ -763,6 +763,27 @@ public void testIssue393() {
     }
 
     @Test
+    public void testPathReference() {
+        SwaggerParser parser = new SwaggerParser();
+        String yaml = "swagger: '2.0'\n" +
+                "info:\n" +
+                "  description: |\n" +
+                "  version: 1.0.0\n" +
+                "  title: testing\n" +
+                "paths:\n" +
+                "   /foo:\n" +
+                "     $ref: 'http://petstore.swagger.io/v2/swagger.json#/paths/~1pet'\n" +
+                "   /bar:\n" +
+                "     $ref: 'http://petstore.swagger.io/v2/swagger.json#/paths/~1pet'\n" +
+                "schemes:\n" +
```

Comment:
This relies on a run-time server being up for the test.  No bueno--will try to change this to use Mocks